gc-policy.yaml

```diff
@@ -1,15 +1,29 @@
+# Production
+testing:
+    cloud-uploads: 2y
+    containers: 2m
+stable:
+    cloud-uploads: 2y
+    containers: 2m
+next:
+    cloud-uploads: 2y
+    containers: 2m
+# Development
 testing-devel:
     cloud-uploads: 2y
     containers: 2w
 next-devel:
     cloud-uploads: 2y
     containers: 2w
```

Comment:
probably should figure out some policy for these `-devel` streams too for images and build

Reply:
though we can do that later